src/ja/common/docker_context.py

```diff
@@ -0,0 +1,88 @@
+"""
+Contains definitions of the various Docker-related attributes of a job.
+These include, but are not limited to, mount points, program environment
+and hardware constraints.
+"""
+from abc import ABC, abstractmethod
+from typing import List
+
+
+class MountPoint:
+    """!
+    A mount point consists of:
+    1. A directory to be mounted(@source_path).
+    2. The target path in the Docker container where the directory needs to be
+    mounted(@mount_path).
+    """
+    def __init__(self, source_path: str, mount_path: str):
+        """!
+        Initializes a new mount point.
+        @source_path The host directory to be mounted.
+        @mount_path The target path in the container to mount at.
```

Comment:
Shouldn't these be "@param source_path", "@param mount_path"?

Reply:
Yes, you're right.